File: CS/RemoveNewButtonInLookups.Module.Web/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace RemoveNewButtonInLookups.Module.Web {
    public sealed partial class RemoveNewButtonInLookupsAspNetModule : ModuleBase {
        public RemoveNewButtonInLookupsAspNetModule() {
            InitializeComponent();
        }
        public override IEnumerable<ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
            return ModuleUpdater.EmptyModuleUpdaters;
        }
    }
}

File: VB/RemoveNewButtonInLookups.Module.Web/Module.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

Namespace RemoveNewButtonInLookups.Module.Web
    Public NotInheritable Partial Class RemoveNewButtonInLookupsAspNetModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()
        End Sub
        Public Overrides Function GetModuleUpdaters(ByVal objectSpace As IObjectSpace, ByVal versionFromDB As Version) As IEnumerable(Of ModuleUpdater)
            Return ModuleUpdater.EmptyModuleUpdaters
        End Function
    End Class
End Namespace
